bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -3536,14 +3579,26 @@
         <comment>Map users to new roles</comment>
         <sql>
             set @id = NULL;
-            select user_id from users where username='admin' into @id;
-            INSERT INTO `user_role` VALUES (@id, 'bypass2FA'),(@id,'Provider'),(@id,'System Developer');
+            select user_id from users where system_id='admin' into @id;
+            INSERT IGNORE INTO `user_role` VALUES (@id,'Provider'),(@id,'System Developer');
 
-            select user_id from users where username='reports-user' into @id;
+            select user_id from users where system_id='superman' into @id;
+            INSERT IGNORE INTO `user_role` VALUES (@id,'Provider'),(@id,'System Developer'), (@id, 'SuperAdmin');
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-201906170327" author="Samuel34">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from users where username = 'reports-user';
+            </sqlCheck>
+            <sqlCheck expectedResult="0">
```

Comment:
I am not sure of this check against the user_role table. This should ideally check whether the role "Reports-App" exists in the roles tables, and then do an "insert  ignore into user_role ... " for given id of "reports-users" and "Reports-App"